projects/plugins/jetpack/to-test.md

```diff
@@ -1,91 +1,41 @@
-- You should be able to transform a child block into any other Form child block
-
-Additionally, when themes do not reset the browser styles, the Jetpack button block will display borders on the frontend. To avoid that, we add our own reset, except when the button uses the "outline" styles, where we do want to show a border:
-
-- Start with a site using the Twenty Twenty Two theme.
-- In a new post, add two blocks that use the Jetpack button block: the Form block as well as the Revue block are good examples.
-- In one of those blocks, use the default "fill" block style, and in the other use the "outline" style.
-- On the frontend, the button should not have a border when using the "fill" block style.
-
-A fix has been added to avoid the following issue: the "Single Choice" and "Multiple Choice" blocks in Jetpack Form blocks, when added to a form inside a Cover block, previously did not obey the background setting set in the editor and would display with a white background. This release fixes the problem, to test:
-
-- Add a cover block to a page, and a form inside it.
-- Add a "Single Choice" or "Multiple Choice" block to the form.
-- Make sure that form blocks always respect the background setting set in the block's settings, regardless of where they are placed.
-- Save changes.
-- Observe no issues on front-end as well.
+### AI Assistant:
+Jetpack 12.2 brings a new AI Assistant block. Given Automattic's current focus on AI, this should be tested. See p1HpG7-mHr-p2 for details.
+
+### My Plan:
+- Check all images are loading.
+- "My Plan" card should no longer show on sites with a free plan but paid add-on features.
+
+### Various block changes:
+- Donations, Paid Content and Payment Buttons blocks are now available with free plans. Check that it works. Information at PCYsg-8Ns-p2#testing-payments may be helpful.
+  - Subscriptions block also has a payment option, see pbNhbs-6Ju-p2#comment-15715
+- Related Posts block can now have a header configured, and there's an author toggle.
+
+### Jetpack Forms:
+Jetpack Forms continues to improve with changes in this release. In particular,
+
+- Make sure things continue to work as expected.
+- Check that the emails sent on a form response look good.
+
+### Image CDN:
+Enable the feature (Jetpack > Settings > Performance > Speed up image load times) and check that image-based features (e.g. Tiled Gallery, Flickr widget, gallery widget, image widget) still work.
+
+### Sharing to Mastodon:
+* Make sure you have a Mastodon account.
+* Make sure your test site has a Social Advanced or Complete subscription, and activate Social (from the My Jetpack section, for instance).
+* Create a new post.
+* Click on the Jetpack menu in the top bar.
+* In the _Share this post_ section, click _Connect an account_.
+* You should be redirected to wordpress.com. Enable the mastodon feature flag by adding the query parameter `flags=mastodon` and reloading the page.
```

Comment:
That flag is no longer necessary. At the moment, you can get the Mastodon UI in Calypso via a blog flag. I assume this will be removed soon. cc @Automattic/jetpack-origin 

```suggestion
* You should be redirected to wordpress.com.
```